app/models/peep.rb

```diff
@@ -0,0 +1,7 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id,   Serial
+  property :peep, String, :length => 0..20, :messages => { :length => 'Your peep can only be 20 characters long'} 
```

Comment:
Metrics/LineLength: Line is too long. [114/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)